packages/next/server/image-optimizer.ts

```diff
@@ -631,6 +631,7 @@ export async function imageOptimizer(
         // will inline a special url to lazily generate the blur placeholder at request time.
         const meta = await getMetadata(optimizedBuffer)
         const opts = {
+          objectFit: undefined, // TODO: what value should go here?
```

Comment:
Need to do some investigation here to test this case. 

I think it won't need objectFit now that I think about it because it can guarantee `blurWidth` and `blurHeight`.